recipes/kickcat/all/conanfile.py

```diff
@@ -0,0 +1,72 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout
+from conan.tools.files import get, copy
+from conan.tools.scm import Version
+import os
+
+required_conan_version = ">=1.55"
+
+
+class KickCATRecipe(ConanFile):
+    name = "kickcat"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/Siviuze/KickCAT"
+    description = "Thin EtherCAT stack designed to be embedded in a more complex software and with efficiency in mind"
+    license = "CeCILL-C"
+    topics = ("ethercat")
```

Comment:
Missing [`package_type`](https://docs.conan.io/2/reference/conanfile/attributes.html#package-type)
```suggestion
    topics = ("ethercat")
    package_type = "library"
```